hr_employee_birth_name/views/hr_employee_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
This header in xml files is not longer needed since some versions ago

Reply:
@pedrobaeza say:
https://github.com/OCA/management-system/pull/471#discussion_r1048396816

Reply:
Yes, stated already in other PRs done today. Not sure why to bother in changing this anyway. You have to reduce the diff to the minimum of not adding a great value.